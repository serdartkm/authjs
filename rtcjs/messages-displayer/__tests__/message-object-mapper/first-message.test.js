import React from 'react'
import FirstMessage from '../../lib/message-object-mapper/first-message'
import renderer from 'react-test-renderer'

describe("SNAPSHOT TEST FirstMessage component snapshot",()=>{
    it("Message local renders correctly",()=>{
        const tree =renderer
        .create(<FirstMessage  FirstMessage datetime={1569393960355} message="First Message" letter ="m" local={true}/>).toJSON()
         expect(tree).toMatchSnapshot();
    })

    it("Message remote renders correctly",()=>{
        const tree =renderer
        .create(<FirstMessage  FirstMessage datetime={1569393960355} message="First Message" letter ="m" local={false}/>).toJSON()
     expect(tree).toMatchSnapshot();
    })
})