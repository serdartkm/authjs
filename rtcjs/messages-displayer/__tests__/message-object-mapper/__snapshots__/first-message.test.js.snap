// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOT TEST FirstMessage component snapshot Message local renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "#FFECB3",
        "borderColor": "#9E9E9E",
        "borderRadius": 15,
        "borderStyle": "solid",
        "borderWidth": 2,
        "margin": 2,
        "maxWidth": "100%",
        "minWidth": "30%",
        "padding": 5,
        "wordBreak": "break-all",
        "wordWrap": "break-word",
      }
    }
  >
    <div>
      First Message
    </div>
    <div
      style={
        Object {
          "fontSize": 10,
          "paddingTop": 2,
          "textAlign": "end",
        }
      }
    >
      <i
        style={
          Object {
            "backgroundColor": "#efebe9",
          }
        }
      >
        11:46:00 AM
      </i>
    </div>
  </div>
</div>
`;

exports[`SNAPSHOT TEST FirstMessage component snapshot Message remote renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "darkSmoke",
        "borderColor": "#80cbc4",
        "borderRadius": 30,
        "borderStyle": "solid",
        "borderWidth": 2,
        "color": "#009688",
        "display": "flex",
        "height": 40,
        "justifyContent": "center",
        "padding": 3,
        "width": 40,
      }
    }
  >
    <div>
      M
    </div>
  </div>
  <div
    style={
      Object {
        "backgroundColor": "#FFECB3",
        "borderColor": "#9E9E9E",
        "borderRadius": 15,
        "borderStyle": "solid",
        "borderWidth": 2,
        "margin": 2,
        "maxWidth": "100%",
        "minWidth": "30%",
        "padding": 5,
        "wordBreak": "break-all",
        "wordWrap": "break-word",
      }
    }
  >
    <div>
      First Message
    </div>
    <div
      style={
        Object {
          "fontSize": 10,
          "paddingTop": 2,
          "textAlign": "end",
        }
      }
    >
      <i
        style={
          Object {
            "backgroundColor": "#efebe9",
          }
        }
      >
        11:46:00 AM
      </i>
    </div>
  </div>
</div>
`;
